plugins/usernames.py

```diff
@@ -56,6 +60,26 @@ async def annika(msg: Message) -> None:
     await msg.reply("enjoy ur italian joke punishment room")
 
 
+@command_wrapper(aliases=("anto", "antonio"))
+async def antonio200509(msg: Message) -> None:
+    db = Database.open("veekun")
+    with db.get_session() as session:
+        species = (
+            session.query(v.PokemonSpeciesNames)
+            .filter_by(local_language_id=8)
+            .order_by(func.random())
+            .first()
+        )
+        if not species:
+            raise SQLAlchemyError("Missing PokemonSpecies data")
+        species_name = species.name
+    text = ""
+    for _ in range(6):
+        text += str(random.randint(0, 9))
+    text = "Antonio" + text + ' guessed "' + species_name + '"!'
```

Comment:
I would probably get rid of the `for` to generate the random number, using something like `str(random.randint(0, 999999)).zfill(6)`.
You should also use f-strings (see [here](https://github.com/Parnassius/cerbottana/blob/eb4a3faa492f04f33857c6dcd3e12949b67efbc7/plugins/usernames.py#L241) for example)